lib/github/github.ex

```diff
@@ -188,6 +201,25 @@ defmodule BorsNG.GitHub do
     user
   end
 
+  @spec get_team_by_name(tconn, bitstring, bitstring) ::
+          {:ok, BorsNG.GitHub.Team.t} | {:error, bitstring}
+  def get_team_by_name(repo_conn, org_name, team_name) do
+    GenServer.call(
+      BorsNG.GitHub,
+      {:get_team_by_name, repo_conn, {org_name, team_name}},
+      Confex.fetch_env!(:bors, :api_github_timeout))
+  end
+
+  @spec is_user_on_team!(tconn, bitstring, integer) ::
+          boolean
+  def is_user_on_team!(repo_conn, username, team_id) do
```

Comment:
I am confused by this. According to Elixir convention, ending with an `!` means that it should throw an error when unable to go through the happy path. If it's returning a bool, it should end with `?`. Also according to Elixir conventions, you don't need to use `is_` when the function ends with `?`, so I'd suggest renaming `user_on_team?` or better yet, `belongs_to_team?` as it would be very fluent based on the context

Reply:
This is a toughie, since I think it'll still throw an error in some scenarios, like if the network fails. But, probably, yeah, the fact that it's also a boolean-returning function should probably take precedence, so it would be named `belongs_to_team?`

Reply:
⚡️